### PR TITLE
feat(react): add custom elements bundle

### PR DESCRIPTION
**Pull request by liamdebeasi**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the new behavior?

Ionic React now uses Stencil's custom elements bundle by default resulting in improved app load times, smaller bundle sizes, and compatibility with tooling such as Vite, ESBuild, and Rollup.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/react/src/components/proxies.ts [hunk 1]
```diff
@@ -3,75 +3,142 @@
 /* auto-generated react proxies */
```
Comment (willmartindev) [inline packages/react/src/components/proxies.ts:3]: Everything below this is still auto generated, correct?

**Review by willmartindev — APPROVED** [review]

File: packages/react/src/components/proxies.ts (hunk 1 above)
Comment (liamdebeasi) [reply packages/react/src/components/proxies.ts:3]: Yes

Comment (willmartindev) [reply packages/react/src/components/proxies.ts:3]: Awesome, everything looks good to me then. Good work 💯

**Comment by liamdebeasi** [comment]
A few things with the latest commits:

1. I got approval from Ely to remove many of the Ionic React + Ionic React Router spec tests as they were either redundant or being skipped altogether.
2. I added a workaround for a CE Build bug where connectedCallback in Stencil Cmp is being fired before the WC is registered

**Review by willmartindev — APPROVED** [review]

**Pull request merged**